### PR TITLE
Fix bug: Update download, add some images

### PR DESCRIPTION
**Pull request by chicky-dev**
https://edu-redmine.framgia.vn/issues/3060

### CODE REVIEW

**Comment by chicky-dev** [comment]
ready

File: app/src/main/java/com/framgia/vhlee/musicplus/ui/home/HomeFragment.java [hunk 1]
```diff
@@ -1,9 +1,15 @@
 package com.framgia.vhlee.musicplus.ui.home;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/vhlee/musicplus/ui/home/HomeFragment.java:2]: remove bl

File: app/src/main/java/com/framgia/vhlee/musicplus/ui/home/HomeFragment.java [hunk 2]
```diff
@@ -134,4 +142,15 @@ public void setImage(ImageView image, String source) {
...
+    @Override
+    public void onClick(View view) {
+        switch (view.getId()) {
+            case R.id.image_cover:
+                break;
+            default:
+                break;
+        }
+    }
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/vhlee/musicplus/ui/home/HomeFragment.java]: remove bl

File: app/src/main/java/com/framgia/vhlee/musicplus/ui/play/PlayActivity.java [hunk 3]
```diff
@@ -171,6 +178,19 @@ public void onClick(View view) {
...
+    @Override
+    public void onClick(DialogInterface dialogInterface, int button) {
+        switch (button) {
+            case DialogInterface.BUTTON_POSITIVE:
+                beginDownload();
+                dialogInterface.dismiss();
+                break;
+            case DialogInterface.BUTTON_NEGATIVE:
+                dialogInterface.dismiss();
+                break;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/vhlee/musicplus/ui/play/PlayActivity.java:203]: add default case

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by chicky-dev** [comment]
ready

File: app/src/main/java/com/framgia/vhlee/musicplus/ui/home/HomeFragment.java (hunk 1 above)
Comment (chicky-dev) [reply app/src/main/java/com/framgia/vhlee/musicplus/ui/home/HomeFragment.java:2]: removed

File: app/src/main/java/com/framgia/vhlee/musicplus/ui/play/PlayActivity.java (hunk 3 above)
Comment (chicky-dev) [reply app/src/main/java/com/framgia/vhlee/musicplus/ui/play/PlayActivity.java:203]: added

**Review by NguyenMinhTuanB — APPROVED** [review]

**Comment by NguyenMinhTuanB** [comment]
conflicted

**Comment by chicky-dev** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**